src/Swarrot/Processor/Retry/README.md

```diff
@@ -22,6 +23,11 @@ in the exchange named `retry` and if the `retry_key_pattern` is
 publish a new message (similar to the first one) in the exchange `retry` with
 the routing_key `key_1`.
 
+If `retry_final_key` is set and `retry_attempts` is reached, the message will be 
+published to the exchange `retry` with the routing key `retry_final_key`.
+If `retry_final_key` is not set and `retry_attempts` is reached, the message will be 
```

Comment:
Can you remove `the message will be lost` please?
It could be true but not always. It depends (a lot) on your own configuration. :)